### PR TITLE
Added namespace for splitting pdfs.

### PR DESCRIPTION
**Pull request by nukecoder**
Had to use ByteArrays to ensure AcroFields were carried over to merged PDF.

### CODE REVIEW

File: .nrepl-port [hunk 1]
```diff
@@ -0,0 +1 @@
+55072
```
Comment (dotemacs) [inline .nrepl-port]: Can this file be removed?

No need to enforce the particular port on any other users.

Comment (nukecoder) [inline .nrepl-port]: Sorry, I didn't know that I had added that to the project. I have deleted
the file from my fork now. Do I need to close the pull request and open a
new one?

Thanks,
Wes

**Comment by dotemacs** [comment]
@nukecoder 

you did it just fine, no need to re-create the pull request.

Let me look at it again a bit later and I'll merge it.

Thank you and @pmn for your work, I like what you've done.

**Comment by dotemacs** [comment]
Thank you @nukecoder your change has now been merged. :)

**Comment by pmn** [comment]
Minor point, would it make sense to increment the version number to make dependency management simpler? Thank you!

**Comment by nukecoder** [comment]
Yeah, I guess it would.

**Comment by dotemacs** [comment]
If you've got time, feel free to create a PR with the version increment, thanks

**Comment by pmn** [comment]
@dotemacs PR created in #12

**Pull request merged**